.github/workflows/run_task.yml

```diff
@@ -0,0 +1,50 @@
+name: Run task in Fargate cluster
+
+on:
+  workflow_call:
+    inputs:
+      container_make_target:
+        description: Make target to execute remotely in a Fargate task
+        required: true
+        type: string
+
+jobs:
+  run_task:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          aws-region: us-east-2
+
+      - name: Set branch name
+        run: |
+          # Short name for current branch. For PRs, use target branch (base ref)
+          GIT_BRANCH="${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}"
+          # Is the ref a tag? If so, remove refs/tags/ prefix
+          GIT_BRANCH="${GIT_BRANCH#refs/tags/}"
+          echo "Git branch is ${GIT_BRANCH}"
+          echo "GIT_BRANCH=$GIT_BRANCH" >> "$GITHUB_ENV"
+
+      - name: Launch Fargate task
+        uses: jsubirat/fargate-runner-action@allow_passing_task_security_group
```

Comment:
I am currently using my fork here since I am waiting for this necessary PR to be merged on the base repo: https://github.com/newrelic/fargate-runner-action/pull/25